src/backend/jag-efiling-api/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.7'
```

Comment:
could we move the `docker-compose` definition to root, so we can root into the repo and go docker-compose up ?